src/components/ReferenceDirectoryWithFilter/index.tsx

```diff
@@ -72,18 +72,26 @@ export const ReferenceDirectoryWithFilter = ({
 
     return categoryData.reduce((acc: FilteredCategoryData[], category) => {
       const filteredSubcats = category.subcats.reduce(
-        (subAcc, subcat) => {
+        (subAcc :typeof category.subcats, subcat) => {
```

Comment:
Generally in the codebase we put the space after the colon when adding a type (e.g. [in this function.](https://github.com/processing/p5.js-website/blob/678a03d2092c3e99e571b7713f8007f7908fe37e/src/layouts/ReferenceLayout.astro#L28)) Could we change it here to match?
```suggestion
        (subAcc: typeof category.subcats, subcat) => {
```